DEVELOPMENT.md

```diff
@@ -19,12 +19,12 @@ You will need to install the following:
 - xsltproc
 - python 2.7
 
-To run end-to-end tests you will also need:
+To run end-to-end (e22) tests you will also need:
```

Comment:
```suggestion
To run end-to-end (e2e) tests you will also need:
```

Reply:
Gah!  Thanks so much for checking that :sweat_smile: